OnceElasticJobListener test case throw Job timeout exception.

## Bug Report

**For English only**, other languages will not accept.

Before report a bug, make sure you have:

- Searched open and closed [GitHub issues](https://github.com/apache/shardingsphere-elasticjob/issues).
- Read documentation: [ElasticJob Doc](https://shardingsphere.apache.org/elasticjob/current/en/overview/).

Please pay attention on issues you submitted, because we maybe need more details. 
If no response anymore and we cannot reproduce it on current information, we will **close it**.

Please answer these questions before submitting your issue. Thanks!

### Which version of ElasticJob did you use?

### Which project did you use? ElasticJob-Lite or ElasticJob-Cloud?

### Expected behavior

### Actual behavior

### Reason analyze (If you can)

### Steps to reproduce the behavior.

### Example codes for reproduce this issue (such as a github link).

error message:

```
------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running org.apache.shardingsphere.elasticjob.lite.integrate.enable.ScheduleEnabledJobIntegrateTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.795 sec - in org.apache.shardingsphere.elasticjob.lite.integrate.enable.ScheduleEnabledJobIntegrateTest
Running org.apache.shardingsphere.elasticjob.lite.integrate.enable.OneOffEnabledJobIntegrateTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.315 sec - in org.apache.shardingsphere.elasticjob.lite.integrate.enable.OneOffEnabledJobIntegrateTest
Running org.apache.shardingsphere.elasticjob.lite.integrate.disable.OneOffDisabledJobIntegrateTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.174 sec - in org.apache.shardingsphere.elasticjob.lite.integrate.disable.OneOffDisabledJobIntegrateTest
Running org.apache.shardingsphere.elasticjob.lite.integrate.disable.ScheduleDisabledJobIntegrateTest
[ERROR] 2020-11-03 10:40:41,798 --50559352531210_test_job_Worker-1-- [org.apache.shardingsphere.elasticjob.error.handler.general.LogJobErrorHandler] Job '50559352531210_test_job' exception occur in job processing
org.apache.shardingsphere.elasticjob.infra.exception.JobSystemException: Job timeout. timeout mills is 100.
	at org.apache.shardingsphere.elasticjob.lite.api.listener.AbstractDistributeOnceElasticJobListener.handleTimeout(AbstractDistributeOnceElasticJobListener.java:106)
	at org.apache.shardingsphere.elasticjob.lite.api.listener.AbstractDistributeOnceElasticJobListener.beforeJobExecuted(AbstractDistributeOnceElasticJobListener.java:75)
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJobFacade.beforeJobExecuted(LiteJobFacade.java:145)
	at org.apache.shardingsphere.elasticjob.executor.ElasticJobExecutor.execute(ElasticJobExecutor.java:100)
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJob.execute(LiteJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
[ERROR] 2020-11-03 10:40:41,912 --50559352531210_test_job_Worker-1-- [org.apache.shardingsphere.elasticjob.error.handler.general.LogJobErrorHandler] Job '50559352531210_test_job' exception occur in job processing
org.apache.shardingsphere.elasticjob.infra.exception.JobSystemException: Job timeout. timeout mills is 100.
	at org.apache.shardingsphere.elasticjob.lite.api.listener.AbstractDistributeOnceElasticJobListener.handleTimeout(AbstractDistributeOnceElasticJobListener.java:106)
	at org.apache.shardingsphere.elasticjob.lite.api.listener.AbstractDistributeOnceElasticJobListener.afterJobExecuted(AbstractDistributeOnceElasticJobListener.java:101)
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJobFacade.afterJobExecuted(LiteJobFacade.java:152)
	at org.apache.shardingsphere.elasticjob.executor.ElasticJobExecutor.execute(ElasticJobExecutor.java:113)
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJob.execute(LiteJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.383 sec - in org.apache.shardingsphere.elasticjob.lite.integrate.disable.ScheduleDisabledJobIntegrateTest
Running org.apache.shardingsphere.elasticjob.lite.internal.snapshot.SnapshotServiceEnableTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.095 sec - in org.apache.shardingsphere.elasticjob.lite.internal.snapshot.SnapshotServiceEnableTest
Running org.apache.shardingsphere.elasticjob.lite.internal.snapshot.SnapshotServiceDisableTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.175 sec - in org.apache.shardingsphere.elasticjob.lite.internal.snapshot.SnapshotServiceDisableTest
Running org.apache.shardingsphere.elasticjob.lite.internal.schedule.JobScheduleControllerTest
^C%
```

--- a/elasticjob-lite/elasticjob-lite-core/src/main/java/org/apache/shardingsphere/elasticjob/lite/api/listener/AbstractDistributeOnceElasticJobListener.java
+++ b/elasticjob-lite/elasticjob-lite-core/src/main/java/org/apache/shardingsphere/elasticjob/lite/api/listener/AbstractDistributeOnceElasticJobListener.java
@@ -53,6 +53,9 @@ public abstract class AbstractDistributeOnceElasticJobListener implements Elasti
     @Override
     public final void beforeJobExecuted(final ShardingContexts shardingContexts) {
         Set<Integer> shardingItems = shardingContexts.getShardingItemParameters().keySet();
+        if (shardingItems.isEmpty()) {
+            return;
+        }
         guaranteeService.registerStart(shardingItems);
         while (!guaranteeService.isRegisterStartSuccess(shardingItems)) {
             BlockUtils.waitingShortTime();
@@ -79,6 +82,9 @@ public abstract class AbstractDistributeOnceElasticJobListener implements Elasti
     @Override
     public final void afterJobExecuted(final ShardingContexts shardingContexts) {
         Set<Integer> shardingItems = shardingContexts.getShardingItemParameters().keySet();
+        if (shardingItems.isEmpty()) {
+            return;
+        }
         guaranteeService.registerComplete(shardingItems);
         while (!guaranteeService.isRegisterCompleteSuccess(shardingItems)) {
             BlockUtils.waitingShortTime();

